frontend/components/InfoField/InfoField.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import { Text, View } from 'react-native';
 import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 
 import GPSInfo from '../GPSInfo/GPSInfo';
 
+import * as actionCreators from './InfoField.Action'
 import { infoFieldStyles } from './InfoField.Styles';
 
 class Info extends React.Component {
+    componentDidMount(){
```

Comment:
The platform request should be in the `Root.js` or `App.js` in this case, every time the view changes back to the information view the platform will be requested...